mon-pix/translations/fr.json

```
@@ -1220,6 +1220,11 @@
       "qrocm": {
         "direction": "Remplissez les champs vides."
       },
+      "recap": {
+        "title": "Bravo ! Module terminé",
+        "backToModuleDetails": "Revenir au détail du module",
```

Retour d' @AnaisAllamand 

```suggestion
        "backToModuleDetails": "Revenir aux détails du module",
```